backend/uclapi/timetable/admin.py

```diff
@@ -0,0 +1 @@
+# Register your models here.
```

Comment:
@Faiz can we remove this file if it's not in use?

Reply:
ook